database_admin/migrate.go

```diff
@@ -2,15 +2,36 @@ package database_admin //nolint:golint,stylecheck
 
 import (
 	"app/base/utils"
+	"database/sql"
 	"fmt"
 	"github.com/golang-migrate/migrate/v4"
-	_ "github.com/golang-migrate/migrate/v4/database/postgres" // postgres database is used
-	_ "github.com/golang-migrate/migrate/v4/source/file"       // we load migrations from local folder
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	_ "github.com/golang-migrate/migrate/v4/source/file" // we load migrations from local folder
+	"github.com/lib/pq"
 	"os"
 )
 
 func MigrateUp(sourceURL, databaseURL string) {
-	m, err := migrate.New(sourceURL, databaseURL)
+	conn, err := pq.NewConnector(databaseURL)
+	conn2 := pq.ConnectorWithNoticeHandler(conn, func(e *pq.Error) {
+		fmt.Printf("Notice: %v\n", e)
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	db := sql.OpenDB(conn2)
+	if _, err = db.Exec("SET client_min_messages TO NOTICE"); err != nil {
+		panic(err)
+	}
+	var driver database.Driver
+	if driver, err = postgres.WithInstance(db, &postgres.Config{}); err != nil {
+		panic(err)
+	}
+
+	m, err := migrate.NewWithDatabaseInstance(sourceURL, utils.GetenvOrFail("DB_NAME"), driver)
+
```

Comment:
Good, it shows Notice messages now. But could you move these new added lines to separate function? E.g. setupMigration (Migrate, err). I think it will make our MigrateUp() function easier to read.

Reply:
Fixed